helpdesk_mgmt_ticket_invoice/models/account_move.py

```diff
@@ -0,0 +1,39 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    ticket_ids = fields.One2many("helpdesk.ticket","invoice_id")
+    count_ticket_ids = fields.Integer(compute="_compute_count_ticket_ids")
+
+    def _compute_count_ticket_ids(self):
+        for record in self:
+            record.count_ticket_ids = record.count_ticket_ids = len(record.ticket_ids)
```

Comment:
Please remove unnecessary code:
```suggestion
            record.count_ticket_ids = len(record.ticket_ids)
```